src/components/MovieInfo/MovieInfo.js

```diff
@@ -1,28 +1,39 @@
 
 import React, { Component } from 'react';
-import testDetails from '../MovieInfo/mockMovieView';
+import { fetchSingleMovieDetails } from '../../apiCalls';
 import './MovieInfo.css';
 
 class MovieInfo extends Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      movie: testDetails.movie
-    };
+  constructor() {
+    super();
+    this.state = {};
   }
 
-  render() {
-    const { movie } = this.state;
+  componentDidMount() {
+    fetchSingleMovieDetails(this.props.selectedMovieId)
+        .then(data => this.setState(data.movie))
+        .catch()
+  }
 
+  render() {
+    let dollarConversion = new Intl.NumberFormat('en-US', {
+      style: 'currency',
+      currency: 'USD'
+    });
     return (
       <section>
-        <img src={movie.poster_path} alt="Movie Poster" />
+        <img src={this.state.poster_path} alt="Movie Poster" />
```

Comment:
Great job remembering to include `alt` tags for img elements.  I'd like to see you refactor this so that each img's alt tag is provide more helpful specific information than just the generic string of "movie poster".  How could you make it dynamic so that the alt tag actually holds the movie's title?